nservicebus/rabbitmq/connection-strings.md

```diff
@@ -29,15 +29,3 @@ For remote host provide username and password because remote hosts don't accept
                           password=mypassowrd"/>
```

Comment:
lol - when I saw the email notification I thought that perhaps *your* password had been commited :stuck_out_tongue_winking_eye: 

Fixed.